fetcher/tar.py

```diff
@@ -99,18 +101,26 @@ def unpack(self, downloadFirst=False):
         update = tarfile.open(name=self.workspacefile, mode='r:*')
         update.extractall(self.workspaceunpack)
```

Comment:
Why not? Isn't the unused rootfs formatted by resinhup at the beginning of the process?